src/Symfony/Bridge/Doctrine/ManagerRegistry.php

```diff
@@ -24,7 +25,20 @@
  */
 abstract class ManagerRegistry extends AbstractManagerRegistry implements ContainerAwareInterface
 {
-    use ContainerAwareTrait;
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @deprecated since version 3.4, to be removed in 4.0 alongside with the ContainerAwareInterface type.
+     */
+    public function setContainer(SymfonyContainerInterface $container = null)
+    {
+        @trigger_error(sprintf('The "%s()" method is deprecated since version 3.4 and will be removed in 4.0. Inject a PSR-11 container using the constructor instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
abstract ManagerRegistry does not accept a container in its argument. only `\Doctrine\Bundle\DoctrineBundle\Registry` does. also https://github.com/doctrine/DoctrineBundle/blob/master/Registry.php#L41 still calls the deprecated method

Reply:
Yes, I'll do the PR next on DoctrineBundle.

Reply:
Let's wait for the PR on DoctrineBundle before merging this one.